lib/matplotlib/axes/_base.py

```diff
@@ -1286,26 +1286,68 @@ def apply_aspect(self, position=None):
                 self.set_xbound((x0, x1))
 
     def axis(self, *v, **kwargs):
-        """
-        Convenience method for manipulating the x and y view limits
-        and the aspect ratio of the plot. For details, see
-        :func:`~matplotlib.pyplot.axis`.
+        """Set axis properties.
+
+        If no positional or kwargs are given, returns the current
+        axes limits are returned with no side effects.
+
+        If a positional argument is passed the keyword arguments are ignored.
+
+        If no positional argument is passed, the keyword arguments are used.
+
+        Parameters
+        ----------
+        v : list of float or {'on', 'off', 'equal', 'tight', 'scaled',\
+         'normal', 'auto', 'image', 'square'}
```

Comment:
The table looks fine to me size wise.  There are a lot of options so it
needs lot of descriptions.

On Sun, Apr 19, 2015 at 5:39 PM basharovV <notifications@github.com> wrote:

> In lib/matplotlib/axes/_base.py
> <https://github.com/matplotlib/matplotlib/pull/4310#discussion_r28657228>:
>
> > -        Convenience method for manipulating the x and y view limits
> > -        and the aspect ratio of the plot. For details, see
> > -        :func:`~matplotlib.pyplot.
>
> axis`.
> > +        """Set axis properties.
> > +
> > +        If no positional or kwargs are given, returns the current
> > +        axes limits are returned with no side effects.
> > +
> > +        If a positional argument is passed the keyword arguments are ignored.
> > +
> > +        If no positional argument is passed, the keyword arguments are used.
> > +
> > +        Parameters
> > +        ----------
> > +        v : list of float or {'on', 'off', 'equal', 'tight', 'scaled',\
> > +         'normal', 'auto', 'image', 'square'}
>
> Yes, the \ puts everything on one line. The only thing is the table size:
> [image: axis_doc]
> <https://cloud.githubusercontent.com/assets/8410950/7221487/d00fd61c-e6ba-11e4-91d2-dbca6173cf28.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4310/files#r28657228>.
>